sphinx/domains/python.py

```diff
@@ -72,35 +72,42 @@ def _parse_arglist(arglist: str) -> addnodes.desc_parameterlist:
     for param in sig.parameters.values():
         if param.kind != param.POSITIONAL_ONLY and last_kind == param.POSITIONAL_ONLY:
             # PEP-570: Separator for Positional Only Parameter: /
-            params += nodes.Text('/')
+            params += addnodes.desc_sig_operator('', '/')
         if param.kind == param.KEYWORD_ONLY and last_kind in (param.POSITIONAL_OR_KEYWORD,
                                                               param.POSITIONAL_ONLY,
                                                               None):
             # PEP-3102: Separator for Keyword Only Parameter: *
-            params += nodes.Text('*')
+            params += addnodes.desc_sig_operator('', '*')
 
         node = addnodes.desc_parameter()
         if param.kind == param.VAR_POSITIONAL:
-            node += nodes.Text('*' + param.name)
+            node += addnodes.desc_sig_operator('', '*')
+            node += addnodes.desc_sig_name('', param.name)
         elif param.kind == param.VAR_KEYWORD:
-            node += nodes.Text('**' + param.name)
+            node += addnodes.desc_sig_operator('', '**')
+            node += addnodes.desc_sig_name('', param.name)
         else:
-            node += nodes.Text(param.name)
+            node += addnodes.desc_sig_name('', param.name)
 
         if param.annotation is not param.empty:
-            node += nodes.Text(': ' + param.annotation)
+            node += addnodes.desc_sig_punctuation('', ':')
+            node += nodes.Text(' ')
+            node += addnodes.desc_sig_name('', param.annotation)
         if param.default is not param.empty:
             if param.annotation is not param.empty:
-                node += nodes.Text(' = ' + str(param.default))
+                node += nodes.Text(' ')
+                node += addnodes.desc_sig_operator('', '=')
+                node += nodes.Text(' ')
             else:
-                node += nodes.Text('=' + str(param.default))
+                node += addnodes.desc_sig_operator('', '=')
+            node += nodes.inline('', param.default, classes=['default_value'])
```

Comment:
I can't find proper toke name from pygments. So I used inline node for default value. Is there good idea?
https://github.com/pygments/pygments/blob/master/pygments/token.py

Reply:
I have no idea...